clang/lib/Headers/__clang_cuda_math.h

```diff
@@ -12,6 +12,10 @@
 #error "This file is for CUDA compilation only."
 #endif
 
+// The __CLANG_GPU_DISABLE_MATH_WRAPPERS macro provides a way to let standard
+// libcalls reach the link step instead of being eagerly replaced.
+#ifndef __CLANG_GPU_DISABLE_MATH_WRAPPERS
```

Comment:
Is this basically equivalent to not including the file at all?

Can you add a Headers test for this

Reply:
Yes, it's unfortunately really difficult to wrangle just this header out of our wrappers however, so it's easier to just override it once. I can make a test for it.